lib/braspag-rest/request.rb

```diff
@@ -0,0 +1,37 @@
+module BraspagRest
+  class Request
+    class << self
+      SALE_ENDPOINT = '/v2/sales/'
+
+      def authorize(request_id, params)
+        gateway_response = RestClient.post(sale_url, params.to_json, default_headers.merge('RequestId' => request_id))
+        BraspagRest::Response.new(gateway_response)
+      rescue RestClient::ExceptionWithResponse => e
+        config.logger.warn("[BraspagRest] #{e}") if config.log_enabled?
```

Comment:
I don't know if this is really necessary, because this exception and the following are only raised by line 7.